### ISSUE TITLE
VSCode autoformat with prettier is not working and it is destroying my productivity

### ISSUE THREAD

**Issue by n1ru4l** [kamilkisiela/graphql-hive#218]
```
["INFO" - 2:46:12 PM] Prettier Options:
{
  "filepath": "/Users/laurinquast/Projects/graphql-code-generator/website/src/components/live-demo/generate.ts",
  "parser": "typescript",
  "trailingComma": "es5",
  "printWidth": 120,
  "singleQuote": true,
  "arrowParens": "avoid",
  "plugins": [
    "prettier-plugin-svelte",
    "prettier-plugin-sh"
  ]
}
["ERROR" - 2:46:12 PM] Error formatting document.
["ERROR" - 2:46:12 PM] Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
Error: Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at _.resolve (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:761)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37165:25
    at Array.map (<anonymous>)
    at Object.load (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37160:65)
    at Object.load [as loadPlugins] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:15932:23)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37227:24
    at Object.Success [as format] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37243:12)
    at t.default.format (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:463:45)
    at t.PrettierEditProvider.provideEdits (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:322:22)
    at M.provideDocumentFormattingEdits (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:137239)
["INFO" - 2:46:12 PM] Formatting completed in 0.006ms.
["INFO" - 2:47:51 PM] Formatting file:///Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
["INFO" - 2:47:51 PM] Using config file at '/Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs'
["INFO" - 2:47:51 PM] Using ignore file (if present) at /Users/laurinquast/Projects/graphql-code-generator/.prettierignore
["INFO" - 2:47:51 PM] File Info:
{
  "ignored": true,
  "inferredParser": null
}
["INFO" - 2:47:51 PM] File is ignored, skipping.
["INFO" - 2:47:57 PM] Formatting file:///Users/laurinquast/Projects/graphql-code-generator/website/src/components/live-demo/generate.ts
["INFO" - 2:47:57 PM] Using config file at '/Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs'
["INFO" - 2:47:57 PM] Using ignore file (if present) at /Users/laurinquast/Projects/graphql-code-generator/.prettierignore
["INFO" - 2:47:57 PM] File Info:
{
  "ignored": false,
  "inferredParser": "typescript"
}
["INFO" - 2:47:57 PM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 2:47:57 PM] Prettier Options:
{
  "filepath": "/Users/laurinquast/Projects/graphql-code-generator/website/src/components/live-demo/generate.ts",
  "parser": "typescript",
  "trailingComma": "es5",
  "printWidth": 120,
  "singleQuote": true,
  "arrowParens": "avoid",
  "plugins": [
    "prettier-plugin-svelte",
    "prettier-plugin-sh"
  ]
}
["ERROR" - 2:47:57 PM] Error formatting document.
["ERROR" - 2:47:57 PM] Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
Error: Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at _.resolve (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:761)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37165:25
    at Array.map (<anonymous>)
    at Object.load (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37160:65)
    at Object.load [as loadPlugins] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:15932:23)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37227:24
    at Object.Success [as format] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37243:12)
    at t.default.format (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:463:45)
    at t.PrettierEditProvider.provideEdits (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:322:22)
    at M.provideDocumentFormattingEdits (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:137239)
["INFO" - 2:47:57 PM] Formatting completed in 0.006ms.
["INFO" - 2:48:12 PM] Formatting file:///Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs
["INFO" - 2:48:12 PM] Using config file at '/Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs'
["INFO" - 2:48:12 PM] Using ignore file (if present) at /Users/laurinquast/Projects/graphql-code-generator/.prettierignore
["INFO" - 2:48:12 PM] File Info:
{
  "ignored": false,
  "inferredParser": "babel"
}
["INFO" - 2:48:12 PM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 2:48:12 PM] Prettier Options:
{
  "filepath": "/Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs",
  "parser": "babel",
  "trailingComma": "es5",
  "printWidth": 120,
  "singleQuote": true,
  "arrowParens": "avoid",
  "plugins": [
    "prettier-plugin-svelte",
    "prettier-plugin-sh"
  ]
}
["ERROR" - 2:48:12 PM] Error formatting document.
["ERROR" - 2:48:12 PM] Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
Error: Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at _.resolve (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:761)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37165:25
    at Array.map (<anonymous>)
    at Object.load (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37160:65)
    at Object.load [as loadPlugins] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:15932:23)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37227:24
    at Object.Success [as format] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37243:12)
    at t.default.format (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:463:45)
    at t.PrettierEditProvider.provideEdits (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:322:22)
    at M.provideDocumentFormattingEdits (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:137239)
["INFO" - 2:48:12 PM] Formatting completed in 0.003ms.
["INFO" - 2:48:15 PM] Formatting file:///Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs
["INFO" - 2:48:15 PM] Using config file at '/Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs'
["INFO" - 2:48:15 PM] Using ignore file (if present) at /Users/laurinquast/Projects/graphql-code-generator/.prettierignore
["INFO" - 2:48:15 PM] File Info:
{
  "ignored": false,
  "inferredParser": "babel"
}
["INFO" - 2:48:15 PM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 2:48:15 PM] Prettier Options:
{
  "filepath": "/Users/laurinquast/Projects/graphql-code-generator/prettier.config.cjs",
  "parser": "babel",
  "trailingComma": "es5",
  "printWidth": 120,
  "singleQuote": true,
  "arrowParens": "avoid",
  "plugins": [
    "prettier-plugin-svelte",
    "prettier-plugin-sh"
  ]
}
["ERROR" - 2:48:15 PM] Error formatting document.
["ERROR" - 2:48:15 PM] Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
Error: Cannot find module 'prettier-plugin-svelte'
Require stack:
- /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js
- /Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at _.resolve (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:761)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37165:25
    at Array.map (<anonymous>)
    at Object.load (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37160:65)
    at Object.load [as loadPlugins] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:15932:23)
    at /Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37227:24
    at Object.Success [as format] (/Users/laurinquast/Projects/graphql-code-generator/node_modules/prettier/index.js:37243:12)
    at t.default.format (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:463:45)
    at t.PrettierEditProvider.provideEdits (/Users/laurinquast/.vscode/extensions/esbenp.prettier-vscode-9.5.0/src/PrettierEditService.ts:322:22)
    at M.provideDocumentFormattingEdits (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:137239)
["INFO" - 2:48:15 PM] Formatting completed in 0.004ms.

```

**Comment by kamilkisiela** [comment]
#242 disabling, until we fix it. The `prettier-plugin-sh` is not really relevant from Hive standpoint.

**Comment by B2o5T** [comment]
I guess @n1ru4l fixed it in 0.0.2 version?

**Comment by kamilkisiela** [comment]
checking
